core-services/prow/02_config/_plugins.yaml

```diff
@@ -1207,6 +1207,12 @@ config_updater:
         - ci
       gzip: true
       name: ci-operator-2.x-configs
+    ci-operator/config/stackrox/stackrox/*-release-4.0*.yaml:
```

Comment:
```suggestion
    ci-operator/config/stackrox/stackrox/*-release-4*.yaml:
```

Reply:
I haven't tried exactly this but I'm afraid this will cause a problem with 4.1 release which will be matched by both this glob and OCP glob
https://github.com/openshift/release/blob/afd229d558d1edb069ecb8eaa6f2faf7ef9c8567/core-services/prow/02_config/_plugins.yaml#L1582-L1587
https://github.com/openshift/release/blob/afd229d558d1edb069ecb8eaa6f2faf7ef9c8567/core-services/prow/02_config/_plugins.yaml#L611-L616

(There still will be a problem with `ci-operator/config/**/*release-4.1.yaml` but I don't want to solve it proactively.)

OSCI rules seem to be that a file can be matched by only one glob. I ran into this and other issues and described my experience in the commit message [`513b6fd` (#37753)](https://github.com/openshift/release/pull/37753/commits/513b6fd858cebdaad6a90afa6ca6c63308119796)